requirements.txt

```diff
@@ -14,7 +14,7 @@ numpy~=1.26.4
 pandas~=2.2.3
 panel~=1.5.2
 photutils~=2.0.0
-pylightcurve~=4.0.1
+pylightcurve==4.0.1
```

Comment:
A Numpy bug was fixed in 4.0.3. Otherwise, there is some concern about using an almost 4-year-old version. Please examine the comments in #1346 ...

Reply:
Yes, it looks like this may be the only option right now. ...